de.wwwtech.ColorMate.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "de.wwwtech.ColorMate",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "colormate",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "build-args": ["--share=network"],
```

Comment:
```suggestion
```
network access is forbidden at build time as the builders are sandboxed. Please have a look at how other Rust apps handle it

Reply:
I wrote a guide on how to do that last year https://belmoussaoui.com/article/8-how-to-flatpak-a-rust-application

Reply:
This is harder than expected. I get a bunch of weird errors.